src/openms/include/OpenMS/ANALYSIS/OPENSWATH/SpectrumExtractor.h

```diff
@@ -110,64 +118,129 @@ namespace OpenMS
 
     void getDefaultParameters(Param& params);
 
+    /**
+      Picks a spectrum's peaks and saves them in picked_spectrum.
```

Comment:
You will need a @brief to start off the description similar to the class doc strings.